spec/components/button_component_spec.rb

```diff
@@ -6,12 +6,11 @@
   let(:content) { 'Button' }
   let(:options) do
     {
-      outline: outline,
       type: type,
     }.compact
   end
 
-  subject(:rendered) { render_inline ButtonComponent.new(options) { content } }
+  subject(:rendered) { render_inline ButtonComponent.new(outline: outline, **options) { content } }
```

Comment:
Would it have been enough to just add the splat and keep all else the same?

```rb
subject(:rendered) { render_inline ButtonComponent.new(**options) { content } }
```

Reply:
It would have, yeah, but my preference was to separate the arguments to match the function